src/lib.rs

```diff
@@ -89,4 +93,21 @@ pub trait UdpStack {
 	fn close(&self, socket: Self::UdpSocket) -> Result<(), Self::Error>;
 }
 
+/// This trait is implemented by UDP/IP stacks.  It provides the ability to
+/// listen for packets on a specified port and send replies.
+pub trait UdpServerStack: UdpStack {
```

Comment:
I think from a clarity perspective, these traits should be `UdpClient` and `UdpServer`.

Reply:
I like these names the best, changed in PR.